documentation/src/docker.md

```diff
@@ -0,0 +1,108 @@
+# Forest in Docker🌲❤️🐋
+
+## Prerequisites
+- Docker engine installed and running. Forest containers are confirmed to run on the following engines:
+  - Docker Engine (Community) on Linux,
+  - Docker for macOS
+  - Podman on WSL
+
+Native images are available for the following platforms:
+- `linux/arm64`
+- `linux/amd64`
+
+The images will work out-of-the box on both Intel processors and macOS with M1/M2.
+
+## Tags
+For the list of all available tags please refer to the [Forest packages](https://github.com/ChainSafe/forest/pkgs/container/forest).
+
+Currently, the following tags are produced:
+- `latest` - latest build of the `main` branch.
+- `date-digest` e.g., `2023-02-17-5f27a62` - all builds that landed on the `main` branch.
+- release tags, available from `v.0.7.0` onwards.
+
+## Usage
+
+### List available flags and/or commands
+```shell
+# daemon
+❯ docker run --init -it --rm ghcr.io/chainsafe/forest:latest --help
+# cli
+❯ docker run --init -it --rm --entrypoint forest-cli ghcr.io/chainsafe/forest:latest --help
+```
+
+### Create a Forest node running calibration network. Then list all connected peers.
+
+```shell
+❯ docker run --init -it --rm --name forest ghcr.io/chainsafe/forest:latest --chain calibnet --auto-download-snapshot
+```
+then in another terminal (sample output)
+```shell
+❯ docker exec -it forest forest-cli net peers
+12D3KooWAh4qiT3ZRZgctVJ8AWwRva9AncjMRVBSkFwNjTx3EpEr, [/ip4/10.0.2.215/tcp/1347, /ip4/52.12.185.166/tcp/1347]
+12D3KooWMY4VdMsdbFwkHv9HxX2jZsUdCcWFX5F5VGzBPZkdxyVr, [/ip4/162.219.87.149/tcp/30141, /ip4/162.219.87.149/tcp/30141/p2p/12D3KooWMY4VdMsdbFwkHv9HxX2jZsUdCcWFX5F5VGzBPZkdxyVr]
+12D3KooWFWUqE9jgXvcKHWieYs9nhyp6NF4ftwLGAHm4sCv73jjK, [/dns4/bootstrap-3.calibration.fildev.network/tcp/1347]
+```
+
+### Use a shared volume to utilise across different Forest images
+Create the volume
+```shell
+docker volume create forest-data
```

Comment:
It looks good so far, could we have an instruction for the desired keystore management as well, see https://github.com/ChainSafe/forest/issues/2554#issuecomment-1434946039

Reply:
I am not sure I get it. At the moment, the shared volume is also sharing the keystore. The desired (with the keystore and wallet moved away from Forest data directory) management should not be a part of the current set of instructions - we want to document what is available now.